vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/GridViewSelectionPage.java

```diff
@@ -33,45 +33,10 @@
 public class GridViewSelectionPage extends LegacyTestView {
 
     public GridViewSelectionPage() {
-        createSingleSelect();
         createMultiSelect();
         createNoneSelect();
     }
 
-    private void createSingleSelect() {
```

Comment:
All single selection tests have been moved, so this setup is not needed anymore.